src/Shared/Recruit.Vacancies.Client/Application/Services/StubNationalMinimumWageService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Services
+{
+    public class StubNationalMinimumWageService : IGetApprenticeshipNationalMinimumWages
```

Comment:
`ApprenticeMinimumWage` (sorry for being pedantic)

Reply:
👍